charts/prometheus-mysql-exporter/templates/_helpers.tpl

```diff
@@ -1,35 +1,54 @@
-{{/* vim: set filetype=mustache: */}}
 {{/*
 Expand the name of the chart.
 */}}
 {{- define "prometheus-mysql-exporter.name" -}}
-{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
-{{- end -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
 
 {{/*
 Create a default fully qualified app name.
 We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
 If release name contains chart name it will be used as a full name.
 */}}
 {{- define "prometheus-mysql-exporter.fullname" -}}
-{{- if .Values.fullnameOverride -}}
-{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" -}}
-{{- else -}}
-{{- $name := default .Chart.Name .Values.nameOverride -}}
-{{- if contains $name .Release.Name -}}
-{{- .Release.Name | trunc 63 | trimSuffix "-" -}}
-{{- else -}}
-{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" -}}
-{{- end -}}
-{{- end -}}
-{{- end -}}
+{{- if .Values.fullnameOverride }}
+{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- $name := default .Chart.Name .Values.nameOverride }}
+{{- if contains $name .Release.Name }}
+{{- .Release.Name | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" }}
+{{- end }}
+{{- end }}
+{{- end }}
 
 {{/*
 Create chart name and version as used by the chart label.
 */}}
 {{- define "prometheus-mysql-exporter.chart" -}}
-{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" -}}
-{{- end -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Common labels
+*/}}
+{{- define "prometheus-mysql-exporter.labels" -}}
+helm.sh/chart: {{ include "prometheus-mysql-exporter.chart" . }}
```

Comment:
Isn't it still better not to include the version in the labels?

Reply:
Why? Whats the benefit?
It's the default helm 3.3.1 uses with "helm create prometheus-mysql-exporter"

Reply:
Clearer Helm upgrade logs maybe?

Reply:
I don't get your point?

Reply:
When you do `helm upgrade` - probably the logs will be cleaner and there will be less shuffling?

Reply:
Not sure if less info in the logs is preferable.

Reply:
If you do an upgrade - should you wanna change only the resources you wanna upgrade? If it's how it is - then you will modify resources that are not really affected.

Reply:
If you raise the chart version for example the deployment is updated but not the pod. So no restart triggered.

Which resources do have you in mind?

Reply:
Yeah, no restart - agreed. Isn't every single resource going to be mentioned in the upgrade logs in this case? That's the only thing that bothers me a little bit because it might be a bit overwhelming though no deal breaker.

Reply:
I'll have to check.
I'm used to fluxcd so i don't look at the logs anyway :D

Reply:
Bad boy! 🥇

Reply:
This is something I'd like to look into